src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/UInt16TypeVisitor.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
```

Comment:
Could you please re-arrange the namespace in the alphabetical order, except the `System.*` namespaces go first? Here's the example:

https://github.com/Azure/azure-functions-openapi-extension/blob/99c208ed4c86a697250f095ce47f293c7b6e2d6d/src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/Int32TypeVisitor.cs#L1-L12

Please do the same to the other five files.

Reply:
Done